### PR TITLE
Update Login page documentation

### PR DESCRIPTION
**Pull request by franciscotco**
Modifications:

 * Update screenshot
 * Add Signup page
 * Add Organization create page
 * Rework text

### CODE REVIEW

File: content/reference/user/login/index.md [hunk 1]
```diff
@@ -6,20 +6,35 @@ lastmod: 2021-03-10T09:29:04-05:00
...
 The GitHub authorization page is opened in a new tab. If no tab opened, make sure that your browser didn't block it.
 {{< /alert >}}
+
+You need to fill the Signup form to create an account on our application.
+
+{{< img src="signup-gatling.png" alt="OAuth GitHub page" >}}
+
+Pressing the **Create account** button to register you on our application. 
+
+If a root administrator added you to his organization, you're logged in.
```
Comment (guilgaly) [inline content/reference/user/login/index.md:31]: I don't think we show the signup form to a user if their account was created be a system admin, do we? So this isn't quite correct (the user won't even see a "create account" button in this case).

File: content/reference/user/login/index.md [hunk 2]
```diff
@@ -6,20 +6,35 @@ lastmod: 2021-03-10T09:29:04-05:00
...
 
 {{< alert danger >}}
 The GitHub authorization page is opened in a new tab. If no tab opened, make sure that your browser didn't block it.
 {{< /alert >}}
+
+You need to fill the Signup form to create an account on our application.
+
+{{< img src="signup-gatling.png" alt="OAuth GitHub page" >}}
+
+Pressing the **Create account** button to register you on our application.
```
Comment (guilgaly) [inline content/reference/user/login/index.md:29]:
```suggestion
{{< img src="signup-gatling.png" alt="OAuth GitHub page" >}}

To register your account, fill in the Signup form and click on the **Create account** button. 
```

File: content/reference/user/login/index.md [hunk 3]
```diff
@@ -6,20 +6,35 @@ lastmod: 2021-03-10T09:29:04-05:00
...
+
+You need to fill the Signup form to create an account on our application.
+
+{{< img src="signup-gatling.png" alt="OAuth GitHub page" >}}
+
+Pressing the **Create account** button to register you on our application. 
+
+If a root administrator added you to his organization, you're logged in.
+
+Otherwise, you're redirected to the **Organization creation** page.
```
Comment (guilgaly) [inline content/reference/user/login/index.md:33]:
```suggestion
Otherwise, you will be redirected to the **Organization creation** page.
```

**Comment by ccedric** [comment]
For the Github authorization screen, can you take the screenshot on cloud.dev?

File: content/reference/user/login/index.md [hunk 4]
```diff
@@ -6,20 +6,35 @@ lastmod: 2021-03-10T09:29:04-05:00
 weight: 10010
 ---
 
-In order to log in, you need an account with the same username as your GitHub account.
+To log in, you need a GitHub account.
 
-If you're the root administrator, we've already created your account.
-
-Otherwise, please reach out to your root administrator to [create your account]({{< ref "../../admin/users#managing-users" >}}).
+If you want to connect with an existing organization, please reach out to your root administrator to [join an organization]({{< ref "../../admin/users#managing-users" >}}).
```
Comment (ccedric) [inline content/reference/user/login/index.md:11]: reach out to a System Admin

File: content/reference/user/login/index.md (hunk 1 above)
Comment (ccedric) [reply content/reference/user/login/index.md:31]: system admin, not root administrator

File: content/reference/user/login/index.md [hunk 5]
```diff
@@ -6,20 +6,35 @@ lastmod: 2021-03-10T09:29:04-05:00
...
+Pressing the **Create account** button to register you on our application. 
+
+If a root administrator added you to his organization, you're logged in.
+
+Otherwise, you're redirected to the **Organization creation** page.
+
+{{< img src="organization-creation.png" alt="OAuth GitHub page" >}}
+
+{{< alert danger >}}
+Creating an organization is irreversible. If you want to leave your organization, contact [support](https://gatlingcorp.atlassian.net/servicedesk/customer/portal/8).
```
Comment (ccedric) [inline content/reference/user/login/index.md:38]: Add a warning on they shouldn't create an organization if they want to join one

File: content/reference/user/login/index.md (hunk 1 above)
Comment (franciscotco) [reply content/reference/user/login/index.md:31]: Done

File: content/reference/user/login/index.md [hunk 6]
```diff
@@ -6,20 +6,35 @@ lastmod: 2021-03-10T09:29:04-05:00
 weight: 10010
```
[the remark this answers is not in the document]
Comment (franciscotco) [reply content/reference/user/login/index.md:6]: Done

File: content/reference/user/login/index.md [hunk 7]
```diff
@@ -6,20 +6,35 @@ lastmod: 2021-03-10T09:29:04-05:00
...
-Otherwise, please reach out to your root administrator to [create your account]({{< ref "../../admin/users#managing-users" >}}).
+If you want to connect with an existing organization, please reach out to your root administrator to [join an organization]({{< ref "../../admin/users#managing-users" >}}).
 
 {{< img src="login-github.png" alt="Login button" >}}
 
-Pressing the **Signup with GitHub** button will redirect you to GitHub, where you will be asked to authorize the application.
+Pressing the **Continue with GitHub** button will redirect you to GitHub, where you will be asked to authorize the application.
 
-You need to **authorize** it. We only ask access to your profile.
+You need to **authorize** it. We only ask access to your profile and email address.
```
[the remark this answers is not in the document]
Comment (franciscotco) [reply content/reference/user/login/index.md:17]: Done

File: content/reference/user/login/index.md (hunk 3 above)
Comment (franciscotco) [reply content/reference/user/login/index.md:33]: Done

**Comment by franciscotco** [comment]
> For the Github authorization screen, can you take the screenshot on cloud.dev?

Image updated

File: content/reference/user/login/index.md (hunk 4 above)
Comment (franciscotco) [reply content/reference/user/login/index.md:11]: Done

File: content/reference/user/login/index.md (hunk 2 above)
Comment (franciscotco) [reply content/reference/user/login/index.md:29]: Done

File: content/reference/user/login/index.md (hunk 5 above)
Comment (franciscotco) [reply content/reference/user/login/index.md:38]: Done

File: content/reference/user/login/index.md [hunk 8]
```diff
@@ -1,25 +1,44 @@
 ---
 title: "Login"
-description: "Learn how to connect to FrontLine with Github."
+description: "Learn how to connect to Gatling Entreprise with Github."
```
Comment (Isammoc) [inline content/reference/user/login/index.md:3]:
```suggestion
description: "Learn how to connect to Gatling Enterprise with GitHub."
```

Comment (franciscotco) [reply content/reference/user/login/index.md]: Done

File: content/reference/user/login/index.md [hunk 9]
```diff
@@ -1,25 +1,44 @@
...
+You need to **authorize** it. We only ask access to your public profile and email address.
 
 {{< img src="login-github-oauth.png" alt="OAuth GitHub page" >}}
 
 {{< alert danger >}}
-The GitHub authorization page is opened in a new tab. If no tab opened, make sure that your browser didn't block it.
+The GitHub authorization page opens in a new tab. If no tab opens, make sure that your browser doesn't block it.
 {{< /alert >}}
+
+If a System Admin added you to his organization, you're logged in.
```
Comment (guilgaly) [inline content/reference/user/login/index.md:25]:
```suggestion
If a System Admin added you to their organization, you're logged in.
```

Comment (franciscotco) [reply content/reference/user/login/index.md]: Done

**Review by ccedric — APPROVED** [review]

**Review by Isammoc — APPROVED** [review]

**Pull request merged**